nupic/frameworks/opf/clamodel.py

```diff
@@ -671,6 +666,12 @@ def _handleCLAClassifierMultiStep(self, patternNZ,
         "No predicted field was enabled! Did you call enableInference()?"
       )
     self._predictedFieldName = predictedFieldName
+    # check if predicted field was correctly specified
+    if not self._predictedFieldName in self._input:
```

Comment:
> should this be rawInput instead of self._input? 

they have the same semantics, I just find `self._input` better as it's used in more places in the model.